### PR TITLE
Mt 215 spacing in accordion

### PR DESCRIPTION
**Pull request by lha047**
Relates to MT-215

### CODE REVIEW

File: stories/Disclosure.stories.svelte [hunk 1]
```diff
@@ -34,6 +34,10 @@
   <section class="content" use:wrapInShadowDom={args.disableCss}>
     <Disclosure title={args.title} loadJs={!args.disableJs}>
       {@html args.body}
+      <ul>
+        <li>punk 1</li>
```
Comment (tajakobsen) [inline stories/Disclosure.stories.svelte:38]: "punk" or "punkt"?

File: src/scss/components/_list.scss [hunk 2]
```diff
@@ -14,6 +14,20 @@ dl {
...
 ol.list-unstyled,
 ul.list-unstyled {
-  list-style: none;
   padding-left: 0;
+  list-style: none;
+}
+
+ul,
+ol,
+ul[role='list'],
```
Comment (tajakobsen) [inline src/scss/components/_list.scss:23]: We will never want `[role='list']` on `ul` or `ol`

File: stories/components/Links.stories.svelte [hunk 3]
```diff
@@ -124,7 +124,7 @@
   </div>
 </Story>
 
-<style lang="scss" global>
+<style lang="scss">
```
Comment (tajakobsen) [inline stories/components/Links.stories.svelte:127]: Is this correct? The scss-style will be scoped to this component now. Maybe it's using the global style from another component in practice now.

**Review by tajakobsen — CHANGES_REQUESTED** [review]
Looks good. I just have a few small questions before we merge.

File: stories/components/Links.stories.svelte (hunk 3 above)
Comment (lha047) [reply stories/components/Links.stories.svelte:127]: Yes it should be scoped to the story component. It's to simulate the centered page layout. If other stories need this they should be customized in those stories

**Review by tajakobsen — APPROVED** [review]
Good job! Closing and merging to main.

**Pull request merged**